core/backport/inc/ROOT/memory.hxx

```diff
@@ -1,4 +1,4 @@
-/// \file ROOT/RMakeUnique.h
+/// \file ROOT/memory.hxx
```

Comment:
Hi @Axel-Naumann, looking at the implementation below, I wonder if we shouldn't try to add a more complete backport, such as the one here: https://isocpp.org/files/papers/N3656.txt. What do you think?

Reply:
I'd say not as part of this PR (which aims at sneaking into v6-12 to not have too many release inconsistencies...).

Apart from that I think we should only implement the minimal parts that we need, to reduce the chance that what we provide is not what the standard says.